src/pages/workspace/accounting/netsuite/export/NetSuiteTaxPostingAccountSelectPage.tsx

```diff
@@ -0,0 +1,82 @@
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {canUseNetSuiteUSATax} = usePermissions();
+
+    const policyID = policy?.id ?? '-1';
+
+    const config = policy?.connections?.netsuite.options.config;
+    const netsuiteTaxAccountOptions = useMemo<SelectorType[]>(() => getNetSuiteTaxAccountOptions(policy ?? undefined, config?.taxPostingAccount), [config?.taxPostingAccount, policy]);
+
+    const initiallyFocusedOptionKey = useMemo(() => netsuiteTaxAccountOptions?.find((mode) => mode.isSelected)?.keyForList, [netsuiteTaxAccountOptions]);
+
+    const {subsidiaryList} = policy?.connections?.netsuite?.options?.data ?? {};
+    const selectedSubsidiary = useMemo(() => {
+        const selectedSub = (subsidiaryList ?? []).find((subsidiary) => subsidiary.internalID === config?.subsidiaryID);
+        return selectedSub;
+    }, [subsidiaryList, config?.subsidiaryID]);
+
+    const updateTaxAccount = useCallback(
+        ({value}: SelectorType) => {
+            if (config?.taxPostingAccount !== value) {
+                Connections.updateNetSuiteTaxPostingAccount(policyID, value, config?.taxPostingAccount);
+            }
+            Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID));
+        },
+        [policyID, config?.taxPostingAccount],
+    );
+
+    const listEmptyContent = useMemo(
+        () => (
+            <BlockingView
+                icon={Illustrations.TeleScope}
+                iconWidth={variables.emptyListIconWidth}
+                iconHeight={variables.emptyListIconHeight}
+                title={translate('workspace.netsuite.noAccountsFound')}
+                subtitle={translate('workspace.netsuite.noAccountsFoundDescription')}
+                containerStyle={styles.pb10}
+            />
+        ),
+        [translate, styles.pb10],
+    );
+
+    return (
+        <SelectionScreen
+            policyID={policyID}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            displayName={NetSuiteTaxPostingAccountSelectPage.displayName}
+            sections={netsuiteTaxAccountOptions.length ? [{data: netsuiteTaxAccountOptions}] : []}
+            listItem={RadioListItem}
+            onSelectRow={updateTaxAccount}
+            initiallyFocusedOptionKey={initiallyFocusedOptionKey}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID))}
+            title="workspace.netsuite.journalEntriesTaxPostingAccount"
+            listEmptyContent={listEmptyContent}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.NETSUITE}
+            shouldBeBlocked={!!config?.suiteTaxEnabled || !config?.syncOptions.syncTax || !canUseTaxNetSuite(canUseNetSuiteUSATax, selectedSubsidiary?.country)}
```

Comment:
I think this should be moved to a utils? I can see it's being used in other pages.